Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -464,6 +468,29 @@ public class JavaScriptLifter: ComponentBase, Lifter {
                 
             case let op as Comment:
                 w.emitComment(op.content)
+
+            case is BeginCodeString:
+                if nestedCodeString > 0 {
```

Comment:
Do you even need this `if` now? 2**0 - 1 is zero, so I think you can even do without